src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  margin: 0 auto;
+  position: relative;
+
+  &__minutes {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -100%);
+    transform-origin: bottom;
+    animation: move 3600s infinite linear;
+  }
+
+  &__seconds {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-100%, -100%);
```

Comment:
The transform property in `.stopwatch__seconds` is set to `translate(-100%, -100%)`. This may cause the seconds hand to be off-center. You might want to adjust this to `translate(-50%, -100%)` to match the `.stopwatch__minutes` class.